### ISSUE TITLE
Add checks for validity of results after use case tiny tests

### ISSUE THREAD

**Issue by cchriste** [SCIInstitute/ShapeWorks#1431]
This is a serious flaw in these tests and one we should address much sooner than later. 
6.2 isn’t too soon imo since if our “tests” all pass, but with incorrect results, we don't catch bugs.

We can save the results and at the end of the run open and compare those files.

**Comment by cchriste** [comment]
I don't think this is a lot of work since the results today should all be correct, so simply saving them and adding a check should be sufficient. I'm not sure what the results even are, so probably not the best person to do this. 

But some of the tests of recent work that only failed on Windows (note: the work itself is robust, but the arrays were handled as if they had been copied, something that now needs to be done explicitly if necessary) would likely have failed everywhere due to incorrect results.

**Comment by akenmorris** [comment]
See also #1063
